<?xml version="1.0" ?>
<launch>
  <arg name="robot_ip"  default="172.16.0.3"/>
  <arg name="trajectory_method" default="1" />
  <arg name="load_gripper" default="false" />
  <include file="$(find franka_control)/launch/franka_control.launch" >
    <arg name="robot_ip" value="$(arg robot_ip)" />
    <arg name="load_gripper" value="$(arg load_gripper)" />
  </include>

  <rosparam command="load" file="$(find franka_aic)/config/franka_aic.yaml" />
  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen"  args="MRAC_controller"/>
<!--   <node pkg="rviz" type="rviz" output="screen" name="rviz" args="-d $(find franka_aic)/launch/robot.rviz"/> -->
<!--   <node name="rqt_reconfigure" pkg="rqt_reconfigure" type="rqt_reconfigure" required="false" /> -->

<!--   <node pkg="rosbag" type="record" args="-o /home/neurorobotic_student/ckk_ws/rosbags/mrac/ -->
<!--     /joint_states -->
<!--     " name="record" output="screen"/> -->

</launch>
